src/3p-frame.js

```diff
@@ -17,14 +17,15 @@
 
 import {assert} from './asserts';
 import {getLengthNumeral} from '../src/layout';
+import {getService} from './service';
 import {documentInfoFor} from './document-info';
 import {getMode} from './mode';
 import {dashToCamelCase} from './string';
-import {parseUrl} from './url';
+import {parseUrl, assertHttpsUrl} from './url';
 
 
 /** @type {!Object<string,number>} Number of 3p frames on the for that type. */
-var count = {};
+const count = {};
```

Comment:
This was just a drive by change. I'm not doing anything with this var here.

This is fine here as long as we don't want to allow toString as a valid 3p embed type :)